### PR TITLE
Dev codeverify

### PR DESCRIPTION
**Pull request by AndrePaulGrenon**
Added some Token verification for the VerifyQrCode. Also added a function to sanitized the code string during code verification. Like for the username modal. Cleaned up some console.log and managed the axios error when triggerd so there are no uncaught errors both in frontend and backend.

### CODE REVIEW

File: app/backend/src/auth/auth.controller.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ const logger = new Logger("AuthController");
 @ApiTags("auth")
 export class AuthController {
   // Here, private means that authService is a member attribute
-  constructor(private authService: AuthService) {}
+  constructor(private authService: AuthService) { }
```
Comment (JohnKingKong) [inline app/backend/src/auth/auth.controller.ts]: lol here I liked the old way better, no space between the empty bracket

File: app/backend/src/auth/auth.service.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ export class AuthService {
   constructor(
     private prisma: PrismaService, // create(), findUnique()
     private tokenClass: TokenIsVerified
-  ) {}
+  ) { }
```
Comment (JohnKingKong) [inline app/backend/src/auth/auth.service.ts]: idem

File: app/backend/src/auth/auth.service.ts [hunk 3]
```diff
@@ -114,49 +114,55 @@ export class AuthService {
...
+
+      const response2 = await axios.get(`${API_42_URL}/v2/me`, {
+        headers: {
+          Authorization: `Bearer ${token.access_token}`
+        }
+      });
+      const userName = response2.data.login + response2.data.id;
+
+      this.tokenClass.tokenStorage.addToken(clientId, token);
+      const theuser = await this.signin({
```
Comment (JohnKingKong) [inline app/backend/src/auth/auth.service.ts:147]: can we change ```theuser``` by ```user```, or smth without the 'the'

File: app/backend/src/auth/auth.controller.ts [hunk 4]
```diff
@@ -27,7 +27,7 @@ const logger = new Logger("AuthController");
 @ApiTags("auth")
 export class AuthController {
   // Here, private means that authService is a member attribute
-  constructor(private authService: AuthService) { }
+  constructor(private authService: AuthService) {}
```
Comment (JohnKingKong) [inline app/backend/src/auth/auth.controller.ts]: HAAA there we go lolol

File: app/backend/src/auth/auth.service.ts [hunk 5]
```diff
@@ -159,9 +158,8 @@ export class AuthService {
         token: token.access_token
       };
       return authEntity;
-    }
-    catch (error) {
-      logger.log("Axios request silenced")
+    } catch (error) {
+      logger.log("Axios request silenced");
```
Comment (JohnKingKong) [inline app/backend/src/auth/auth.service.ts:162]: use ```logger.debug``` here pls

**Review by JohnKingKong — CHANGES_REQUESTED** [review]
Overall pretty dope, just naming stuff, and change one single ```logger.log``` to ```logger.debug```

**Review by oddtiming — APPROVED** [review]
Fixed issues raised by @JohnKingKong

**Review by JohnKingKong — APPROVED** [review]
*Chef's kiss*

File: app/backend/src/auth/auth.service.ts (hunk 5 above)
Comment (oddtiming) [reply app/backend/src/auth/auth.service.ts]: done

**Pull request merged**